pkgs/development/python-modules/pre-commit-hook-ensure-sops/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, applyPatches, fetchFromGitHub, ruamel-yaml }:
+
+buildPythonPackage rec {
+  pname = "pre-commit-hook-ensure-sops";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "yuvipanda";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-8sMmHNzmYwOmHYSWoZ4rKb/2lKziFmT6ux+s+chd/Do=";
+  };
+
+  # Add the command-line entrypoint to pyproject.toml
+  patches = [ ././entrypoint.patch ];
+
+  propagatedBuildInputs = [ ruamel-yaml ];
```

We expand lists like that.

```suggestion
  propagatedBuildInputs = [
    ruamel-yaml
  ];
```